torchrl/envs/common.py

```diff
@@ -1933,6 +1933,17 @@ def state_spec_unbatched(self, spec: Composite):
         spec = spec.expand(self.batch_size + spec.shape)
         self.state_spec = spec
 
+    def _skip_tensordict(self, tensordict):
+        # Creates a "skip" tensordict, ie a placeholder for when a step is skipped
+        next_tensordict = self.full_done_spec.zero()
+        next_tensordict.update(self.full_observation_spec.zero())
+        next_tensordict.update(self.full_reward_spec.zero())
+        # Copy the data from tensordict in `next`
+        next_tensordict.update(
+            tensordict.select(*next_tensordict.keys(True, True), strict=False)
```

Comment:
It seems that this line will not select any `NonTensorData` in `next_tensordict`, with the reason being that this line yields nothing I believe

https://github.com/pytorch/tensordict/blame/7dd385b96dc0a75a851ca8a100f4765791cbd7aa/tensordict/_td.py#L4473

Is this by design?